app/javascript/components/student/solution-summary/Header.tsx

```diff
@@ -147,33 +179,34 @@ export const Header = ({
           : '',
       ].filter((comment) => comment.length > 0)
 
-      if (isConceptExercise) {
-        return (
-          <header>
-            <div className="info">
-              <h2>Your solution is good enough to continue!</h2>
-              <p>
-                We’ve analysed your solution and have {toSentence(comments)}.
-                You can either continue or address the recommendations first -
-                your choice!
-              </p>
-            </div>
-            <div className="status passed">Tests Passed</div>
-          </header>
-        )
-      } else {
-        return (
-          <header>
-            <div className="info">
-              <h2>Your solution worked, but you can take it further…</h2>
-              <p>
-                We’ve analysed your solution and have {toSentence(comments)}. We
-                suggest addressing the recommendations before proceeding.
-              </p>
-            </div>
-            <div className="status passed">Tests Passed</div>
-          </header>
-        )
+      switch (exerciseType) {
+        case 'concept':
+          return (
+            <header>
+              <div className="info">
+                <h2>Your solution is good enough to continue!</h2>
+                <p>
+                  We’ve analysed your solution and have {toSentence(comments)}.
+                  You can either continue or address the recommendations first -
```

Comment:
Should we pluralize `recommendations`? There could be just one.

Reply:
I've added an improvement issue for this :+1: